src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
If you sent an image, I don't think they work through github over email.

If it helps, note that all catalogs for all frames are combined together at the start of processing.